variables.tf

```diff
@@ -26,6 +26,12 @@ variable "cpu_utilization_alarm_parameters" {
   default     = { create_alarm = true, datapoints_to_alarm = 6, evaluation_periods = 6, period = 300, statistic = "Maximum", threshold = 90.0 }
 }
 
+variable "create_cloudwatch_agent_alarms" {
+  type        = bool
+  description = "A Boolean value indicating whether or not to create alarms based on CloudWatch metrics written by the CloudWatch Agent.  You will generally want this variable to be set to true."
```

Comment:
It might be more helpful to mention that it should be `false` when there's no guarantee of the CloudWatch Agent running on the instance.

Reply:
Please see commit 63d38f0.